### PR TITLE
Add SW_NO_UPSTREAM_REAL_ADDRESS config doc

### PR DESCRIPTION
**Pull request by wu-sheng**
Exit spans with the component in the list would not generate the client-side instance relation metrics. As some tracing plugins can't collect the real peer IP address, such as Nginx-LUA and Envoy.

### CODE REVIEW

**Review by kezhenxu94 — COMMENTED** [review]
Does it really work? `noUpstreamRealAddressAgents` is `final` and unmodifiable list (`Collections.singletonList(6000)`)
 https://github.com/apache/skywalking/blob/4f1dab70037428d1cd09874acee1887fe46e0078/oap-server/analyzer/agent-analyzer/src/main/java/org/apache/skywalking/oap/server/analyzer/provider/AnalyzerModuleConfig.java#L47

**Comment by wu-sheng** [comment]
The config didn't work from the 8.0 or 8.1. Actually, the backend initializer doesn't support list or map :) I just refactored the codes.

File: docs/en/setup/backend/configuration-vocabulary.md [hunk 1]
```diff
@@ -138,6 +138,7 @@ core|default|role|Option values, `Mixed/Receiver/Aggregator`. **Receiver** mode
 | - | - |slowDBAccessThreshold|The slow database access thresholds. Unit ms.|SW_SLOW_DB_THRESHOLD|default:200,mongodb:100|
 | - | - |forceSampleErrorSegment|When sampling mechanism activated, this config would make the error status segment sampled, ignoring the sampling rate.|SW_FORCE_SAMPLE_ERROR_SEGMENT|true|
 | - | - |segmentStatusAnalysisStrategy|Determine the final segment status from the status of spans. Available values are `FROM_SPAN_STATUS` , `FROM_ENTRY_SPAN` and `FROM_FIRST_SPAN`. `FROM_SPAN_STATUS` represents the segment status would be error if any span is in error status. `FROM_ENTRY_SPAN` means the segment status would be determined by the status of entry spans only. `FROM_FIRST_SPAN` means the segment status would be determined by the status of the first span only.|SW_SEGMENT_STATUS_ANALYSIS_STRATEGY|FROM_SPAN_STATUS|
+| - | - |noUpstreamRealAddressAgents|Exit spans with the component in the list would generate the client-side instance relation metrics. As some tracing plugins can't collect the real peer ip address, such as Nginx-LUA and Envoy. |SW_NO_UPSTREAM_REAL_ADDRESS|6000,9000|
```
Comment (kezhenxu94) [inline docs/en/setup/backend/configuration-vocabulary.md:141]:
```suggestion
| - | - |noUpstreamRealAddressAgents|Exit spans with the component in the list would not generate the client-side instance relation metrics. As some tracing plugins can't collect the real peer ip address, such as Nginx-LUA and Envoy. |SW_NO_UPSTREAM_REAL_ADDRESS|6000,9000|
```

File: oap-server/server-bootstrap/src/main/resources/application.yml [hunk 2]
```diff
@@ -181,6 +181,9 @@ agent-analyzer:
     slowDBAccessThreshold: ${SW_SLOW_DB_THRESHOLD:default:200,mongodb:100} # The slow database access thresholds. Unit ms.
     forceSampleErrorSegment: ${SW_FORCE_SAMPLE_ERROR_SEGMENT:true} # When sampling mechanism active, this config can open(true) force save some error segment. true is default.
     segmentStatusAnalysisStrategy: ${SW_SEGMENT_STATUS_ANALYSIS_STRATEGY:FROM_SPAN_STATUS} # Determine the final segment status from the status of spans. Available values are `FROM_SPAN_STATUS` , `FROM_ENTRY_SPAN` and `FROM_FIRST_SPAN`. `FROM_SPAN_STATUS` represents the segment status would be error if any span is in error status. `FROM_ENTRY_SPAN` means the segment status would be determined by the status of entry spans only. `FROM_FIRST_SPAN` means the segment status would be determined by the status of the first span only.
+    # Nginx and Envoy agents can't get the real remote address.
+    # Exit spans with the component in the list would generate the client-side instance relation metrics.
```
Comment (kezhenxu94) [inline oap-server/server-bootstrap/src/main/resources/application.yml:185]:
```suggestion
    # Exit spans with the component in the list would not generate the client-side instance relation metrics.
```

**Review by kezhenxu94 — APPROVED** [review]
Looks good

**Review by ascrutae — APPROVED** [review]

**Pull request merged**